src/lib/components/Tutorials.svelte

```diff
@@ -78,11 +78,17 @@
 
   <div class="tutorials--list-container">
     <div class="tutorials--list">
-      {#each tutorials as { title, slug, icon, service }}
-        {#if selectedFilters[service] || noFilterSelected}
-          <a href={`${base}/tutorials/${slug}`} class="tutorial--link">
-            <SvgIcon class="tutorial--icon" src={findIconSrc(icon)} />
-            <div class="tutorial--title">{title}</div>
+      {#each tutorials as tutorial}
```

Comment:
La dénormalisation ne fonctionnait pas ?

```suggestion
      {#each tutorials as { title, slug, icon, service, url }}
      ...
          <a
            href={url ?? `${base}/tutorials/${tutorial.slug}`}
            target={url ? "_blank" : "_self"}
            class="tutorial--link"
          >
```
```

Reply:
C'est par rapport au type de Tutorial, il peut soit être InternalTutorial ou ExternalTutorial dans l'un url existe mais pas l'autre du coup au lieu de s'embêter à dire le type on fait pas de dénormalisation, c'est pas très coûteux je trouve

Reply:
Dans un cas l'URL sera undefined et dans l'autre non. Je ne vois le problème. Je vais regarder